### DISCUSSION TITLE
please can you give the updated project to Codeigniter 4 and thanks in advance

### DISCUSSION THREAD

**Discussion by kingdov** [ronknight/InventorySystem#33] [Q&A] [answered]
I try to use the current project but is with old codeigniter version and I get tow much errors , so if anyone already have the updated version I hope if he can share it with us and thanks in advance

**Comment by ronknight** [comment, marked as answer]
We are looking to update this. Please be patient.

**Comment by Debashish3530** [comment]
Can you please help me to execute this code
